boards/arm/thingy91_nrf9160/board_nonsecure.c

```diff
@@ -29,11 +30,23 @@ static int thingy91_magpio_configure(void)
 		return 0;
 	}
 
+	err = nrf_modem_lib_get_init_ret();
+	if (err < 0) {
+		LOG_ERR("nrf_modem_lib_get_init_ret failed, error: %d", err);
+		return err;
+	} else if (err > 0) {
+		LOG_WRN("A modem firmware upgrade has been performed, reboot is expected.");
+
+		/** If a modem firmware upgrade has been performed, calls to nrf_modem_at_
+		 *  is expected to fail until a reboot has been carried out.
```

Comment:
`are expected`

Reply:
@simensrostad Will you correct? Or merge as is.

Reply:
Fixed the comment.